.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "kraftvaerk",
+  "env": {
+    "jest": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2017
```

Comment:
Had to add this in order to use `async` & `await`. Probably we better specify it in our ESLint config...